src/components/ContextBreadcrumbs.tsx

```diff
@@ -123,7 +125,17 @@ const BreadCrumb = React.memo(
           ) : label === HOME_TOKEN ? (
             <HomeLink color='gray' size={16} className={css({ position: 'static' })} />
           ) : (
-            <Link className='extend-tap-small' simplePath={simplePath} label={label} />
+            <Link
+              cssRaw={css.raw(
+                {
+                  margin: '-0.25em',
+                  padding: '0.25em',
+                },
```

Comment:
Since we already have the `extendTap` recipe, let's make a `small` variant for this case.